go.mod

```diff
@@ -1,39 +1,73 @@
 module code.cloudfoundry.org/grootfs
 
-go 1.16
+go 1.17
```

Comment:
Is there a reason we only bumped to golang `1.17`? This property always trips people up and we talked about making it consistent with what is used in the release.

Reply:
It was a requirement for the other version bumps. The newer packages require it.